spyder/config/main.py

```diff
@@ -57,10 +57,13 @@
 # Run cell shortcuts
 if sys.platform == 'darwin':
     RUN_CELL_SHORTCUT = 'Meta+Return'
+    RE_RUN_LAST_CELL_SHORTCUT = 'Meta+Shift+R'
 else:
     RUN_CELL_SHORTCUT = 'Ctrl+Return'
+    RE_RUN_LAST_CELL_SHORTCUT = 'Ctrl+Shift+R'

```

Comment:
`Ctrl+Shift+...` is reserved to give focus to plugins. Please change this one to `Alt+Shift+R`